geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/ingest/HBaseIngestCommand.scala

```diff
@@ -27,6 +27,7 @@ class HBaseIngestCommand extends IngestCommand[HBaseDataStore] with HBaseDataSto
 
   override def libjarsPaths: Iterator[() => Seq[File]] = Iterator(
     () => ClassPathUtils.getJarsFromEnvironment("GEOMESA_HBASE_HOME"),
+    () => ClassPathUtils.getJarsFromEnvironment("GEOMESA_HOME"),
```

Comment:
We have deprecated the raw `GEOMESA_HOME` in favor of distribution specific ones (e.g. `GEOMESA_HBASE_HOME`). Did you need this to get it to work? Possibly your setup was incorrect...